README.md

```diff
@@ -2,51 +2,58 @@
 
 This repository contains the configuration files the Kubernetes deployments for some of [our websites](https://github.com/canonical-websites).
 
+We are using [jinja2](https://jinja.palletsprojects.com/) as a template system for our configurations. All the template values for our projects can be found under the sites folder.
+
 ## Structure
 
 We deploy our services into one of two namespaces:
 
 - `staging`: Services for our staging servers (there's usually one per production website)
 - `production`: Services for our production servers
 
-### `services`
-
-Both the [service](https://kubernetes.io/docs/concepts/services-networking/service/) and [deployment](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/) config for a specific service are stored together in a single file at:
-
-> `services/{primary-domain}.yaml`
-
-E.g. the service and deployment settings for https://snapcraft.io are in `services/snapcraft.io.yaml`.
+### Template system
+All configurations for our projects can be found under the sites folder. Using the konf.py script, we generate the following Kubernetes objects:
+- service
+- deployment
+- ingress
 
-This config is then used to configure `snapcraft.io` services in both the `staging` and `production` namespace.
-
-### `ingresses`
+It is possible to generate only some of these objects, E.g.:
+``` bash
+./konf.py staging sites/canonical-com.yaml --templates ingress
```

Comment:
Is this still possible? We discussed changing this.

Reply:
Oh, I didn't update this README.md, I will update it soon

Reply:
No rush. It's gonna take me a while to properly review this.